h/assets.yaml

```diff
@@ -214,6 +214,26 @@ inject_css:
         - h:static/styles/variables.scss
         - h:static/styles/mixins/icons.scss
 
+# the H website
+site_js:
+  output: scripts/site-bundle.js
```

Comment:
Okay, that's fine for now. I'm not entirely sure we even need to be minifying/obfuscating our scripts, because we serve them gzipped, so the delivered size is barely improved. Stripping comments is still valuable, though, so ¯\_(ツ)_/¯. We can revisit this in future.